.github/workflows/deploy_tre.yml

```diff
@@ -44,53 +38,167 @@ jobs:
           ARM_TENANT_ID: "${{ secrets.ARM_TENANT_ID }}"
           ARM_CLIENT_ID: "${{ secrets.ARM_CLIENT_ID }}"
           ARM_CLIENT_SECRET: "${{ secrets.ARM_CLIENT_SECRET }}"
           ARM_SUBSCRIPTION_ID: "${{ secrets.ARM_SUBSCRIPTION_ID }}"
+          RESOURCE_LOCATION: "${{ secrets.LOCATION }}"
+          RESOURCE: "${{ secrets.API_CLIENT_ID }}"
+          TRE_ID: "${{ secrets.TRE_ID }}"
+          TRE_URL:
+            "https://${{secrets.TRE_ID}}.${{secrets.LOCATION}}.cloudapp.azure.com"
+          TF_VAR_location: ${{ secrets.LOCATION }}
           TF_VAR_tre_id: ${{ secrets.TRE_ID }}
           TF_VAR_terraform_state_container_name:
             ${{ secrets.TF_STATE_CONTAINER }}
-          TF_VAR_mgmt_resource_group_name:
-            ${{ secrets.MGMT_RESOURCE_GROUP }}
+          TF_VAR_mgmt_resource_group_name: ${{ secrets.MGMT_RESOURCE_GROUP }}
           TF_VAR_mgmt_storage_account_name:
             ${{ secrets.STATE_STORAGE_ACCOUNT_NAME }}
-          TF_VAR_location: ${{ secrets.LOCATION }}
-          TF_VAR_acr_name: ${{ secrets.ACR_NAME }}
           TF_VAR_core_address_space: ${{ secrets.CORE_ADDRESS_SPACE }}
           TF_VAR_tre_address_space: ${{ secrets.TRE_ADDRESS_SPACE }}
           TF_VAR_swagger_ui_client_id: "${{ secrets.SWAGGER_UI_CLIENT_ID }}"
           TF_VAR_aad_tenant_id: "${{ secrets.AAD_TENANT_ID }}"
           TF_VAR_api_client_id: "${{ secrets.API_CLIENT_ID }}"
           TF_VAR_api_client_secret: "${{ secrets.API_CLIENT_SECRET }}"
-          TF_VAR_api_app_service_plan_sku_tier: "PremiumV3"
-          TF_VAR_api_app_service_plan_sku_size: "P1v3"
-          TF_VAR_debug: false
-          TF_VAR_resource_processor_type: "vmss_porter"
-          TF_VAR_deploy_gitea: true
-          TF_VAR_deploy_nexus: true
-          DEPLOY_NEXUS: true
-          ACR_NAME: ${{ secrets.ACR_NAME }} # scripts are using this too
-          LOCATION: ${{ secrets.LOCATION }} # scripts are using this too
-          TRE_URL: "https://${{secrets.TRE_ID}}.${{secrets.LOCATION}}.cloudapp.azure.com"
-          TRE_ID: "${{secrets.TRE_ID}}"
+          TF_VAR_acr_name: ${{ secrets.ACR_NAME }}
+
+      - name: Notify dedicated teams channel
+        uses: sachinkundu/ms-teams-notification@1.4
+        if: ${{ failure() && github.ref == 'refs/heads/main' }}
+        with:
+          github-token: ${{ github.token }}
+          ms-teams-webhook-uri: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
+          notification-summary: "Deploy TRE Failed"
+          notification-color: dc3545
+          timezone: Europe/Zurich
+
+      - name: Push cached devcontainer
+        if: ${{ success() }}
+        shell: bash
         run: |
-          export TF_VAR_arm_subscription_id=$ARM_SUBSCRIPTION_ID
+          set -e
+          docker image tag tredev:latest ${{ secrets.ACTIONS_ACR_NAME }}.azurecr.io/tredev:latest
+          docker image push ${{ secrets.ACTIONS_ACR_NAME }}.azurecr.io/tredev:latest
 
-          make all
+  build_docker_images:
```

Comment:
This job outputs warnings I think we should try to avoid. Not sure exactly where it comes from...

> Build Docker Images (build-and-push-gitea)
error: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/buildx_buildkit_builder-19ce2ecb-5976-4003-a284-c3fac728017b0/json": dial unix /var/run/docker.sock: connect: permission denied

Reply:
I've removed the installation of Buildkit using a Docker action and used it via ENV vars and the warning has gone away